examples/readme_examples.rs

```diff
@@ -3,9 +3,9 @@ extern crate palette;
 extern crate num;
 
 use image::{RgbImage, GenericImage};
-use num::traits::Float;
+use num::Float;
 
-use palette::{Rgba, Gradient, Mix};
+use palette::{Rgba, Gradient, Mix, flt};
```

Comment:
Haha, it was almost flawless :smile: This is why it's nice to have a second pair of eyes that can review ones changes. I don't know if you have considered it, but you are most welcome to review my pull requests, as well. You have full permission to use my own rules and guidelines against me :wink: